vendor-onboarding-checklist.md

```diff
@@ -0,0 +1,175 @@
+# Vendor onboarding checklist
+
+## Purpose
+
+This document supports the Division of Public Assistance Eligibility
+Information System Replacement (EIS-R) Project. Specifically, this
+document addresses our agile acquisition practice and onboarding new
+vendor staff. Within agile acquisition, we anticipate bringing in a
+potentially different vendor every three to four months, possibly more
+often when we begin running multiple acquisitions simultaneously. With
+short timelines, this onboarding checklist will bring new vendor staff
+up to speed as quickly as possible.
+
+# Audience
+
+This document intends to inform EIS-R State of Alaska project staff and
+the vendors we engage with in various product-increment acquisitions.
+
+# Roles
+
+This section identifies the anticipated vendor roles that will be
+engaged with the
+project.
+
+| **Role**                 | **Description**                                                                             |
+| ------------------------ | ------------------------------------------------------------------------------------------- |
+| Project manager          | Manage the project                                                                          |
+| Technical lead           | Lead the technical work                                                                     |
+| Developers               | Develop client and server application components using CSS, HTML5, Javascript and .NET Core |
+| Researcher               |                                                                                             |
+| User experience designer | Design the user experience                                                                  |
+| Visual designer          |                                                                                             |
+| Scrum Master             | Lead internal daily scrum sessions                                                          |
+
+# Tools and resources
+

```

Comment:
Review team in the DevSecOps road show weekly recommended we pull out common items to help reduce duplication and clarify what unique access is granted to different roles.

Reply:
Done.

Reply:
The "Tools and Resources" list is also pretty wet.
As in the opposite of DRY.
(Don't Repeat Yourself) :)